src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -784,6 +778,142 @@ public void orderByAscFieldWithSpaceTest() throws IOException {
         Assert.assertEquals("The list is not in ascending order", sortedTestFields, testFields);
     }
 
+    @Test
+    public void testWhereWithBoolEqualsTrue() throws IOException {
+        JSONObject response = executeQuery(
+                    StringUtils.format("SELECT * " +
+                                    "FROM %s/account " +
+                                    "WHERE male = true ORDER BY age LIMIT 5",
+                            TestsConstants.TEST_INDEX_BANK)
+        );
+
+        JSONArray hits = getHits(response);
+        Assert.assertThat(hits.length(), equalTo(4));
+    }
+
+    @Test
+    public void testWhereWithBoolEqualsTrueAndGroupBy() throws IOException {
+        JSONObject response = executeQuery(
+                String.format(Locale.ROOT, "SELECT * " +
+                                "FROM %s/account " +
+                                "WHERE male = true GROUP BY balance LIMIT 5",
+                        TestsConstants.TEST_INDEX_BANK)
+        );
+
+        JSONObject testAgg = (response.getJSONObject("aggregations")).getJSONObject("balance");
```

Comment:
remove unnecessary parentheses.

You also can make it a bit simpler by
(JSONArray)response.query("/aggregations/balance/buckets")